tests/test_transforms/test_decomposition/test_change_points_based/test_change_points_models/test_base_change_points_model.py

```diff
@@ -0,0 +1,20 @@
+import pandas as pd
+
+from etna.transforms.decomposition.change_points_based.change_points_models import BaseChangePointsModelAdapter
+
+
+def test_build_intervals():
+    """Check correctness of intervals generation with list of change points."""
+    change_points = [pd.Timestamp("2020-01-01"), pd.Timestamp("2020-01-18"), pd.Timestamp("2020-02-24")]
```

Comment:
Create fixture for this

Reply:
this test is not mine and it does not relates to the issue

I just moved it to another file so I don't think it's ok to change it in this pr